index.html

```diff
@@ -0,0 +1,234 @@
+                    </p>
+                    
+                    <div class="social_media">
+                        <h4>LET'S CONNECT</h4>
+                        <ul>
+                            <li>
+                                <a href="#">
+                                    <img src="img/Vector4.png" alt="github picture">
+                                </a>
+                            </li>
+                        </ul>
+                        <ul>
+                            <li>
+                                <a href="#">
+                                    <img src="img/Vector2.png" alt="linkedin picture">
+                                    <img src="img/Vector3.png" alt="linked picture">
+                                </a>
+                            </li>
+                        </ul>
+                        <ul>
+                            <li>
+                                <a href="#">
+                                    <img src="img/Vector5.png" alt="peace symbol">
+                                </a>
+                            </li>
+                        </ul>
+                        <ul>
+                            <li>
+                                <a href="#">
+                                    <img src="img/Vector.png" alt="twitter picture">
+                                </a>
+                            </li>
+                        </ul>
+                        <ul>
+                            <li>
+                                <a href="#">
+                                    <img src="img/Vector1.png"alt="Gmail picture">
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+
+                    <button type="button" class="get">Get my resume</button>
+
+                    <ul>
+                        <li>
+                            <div class="languages-skills">
+                                <img src="img/Frame 7.png">
+                            </div>
+                        </li>
+                        <li>
+                            <div class="my-skills">
+                                <img src="img/Frame 9.png" alt="javascript picture">
+                                <img src="img/Frame 10.png" alt="html picture">
+                                <img src="img/Frame 11.png" alt="css picture">
+                            </div>
+                        </li>
+                        <li>
+                            <div class="languages-skills">
+                                <img src="img/Frame 8.png">
+                            </div>
+                        </li>
+                        <li>
+                            <div class="languages-skills">
+                                <img src="img/Frame 9 (1).png">
+                            </div>
+                        </li>
+                    </ul>
+                </section>
+        </section>
```

Comment:
- [ ] Kindly make sure the `About myself` section contents are aligned on the page by using ` CSS `flexbox` in order to meet the project requirements. It will even help in the future when it comes to making your website responsive across many screens. :wink: